src/lib/ibcclient.ts

```diff
@@ -1368,13 +1382,18 @@ export interface CreateClientArgs {
   consensusState: TendermintConsensusState;
 }
 
+// this will query for the unbonding period.
+// if the trusting period is not set, it will use 2/3 of the unbonding period
 export async function buildCreateClientArgs(
   src: IbcClient,
-  unbondingPeriodSec: number,
-  trustPeriodSec: number
+  trustPeriodSec?: number
 ): Promise<CreateClientArgs> {
   const header = await src.latestHeader();
   const consensusState = buildConsensusState(header);
+  const unbondingPeriodSec = await src.getUnbondingPeriod();
+  if (trustPeriodSec === undefined) {
+    trustPeriodSec = Math.floor((unbondingPeriodSec * 2) / 3);
```

Comment:
This will be 0 for very small `unbondingPeriodSec` values, is that intended?

Reply:
These numbers are typically in weeks and the shortest I have ever seen is hours. I don't think less can be used with IBC.